pkg/db/level/state_computer.go

```diff
@@ -0,0 +1,92 @@
+package level
```

Comment:
not sure how helpful it would be since we'll need to mock a bunch of things out, but do we want to test this package?